api/api.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"io"
+	"net/http"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/jdxcode/netrc"
+)
+
+var (
+	// BaseURL is the root of the Section API
+	BaseURL = "https://aperture.section.io/api/v1"
+)
+
+func getBasicAuth() (u, p string, err error) {
+	usr, err := user.Current()
+	if err != nil {
+		panic(err)
+	}
+	n, err := netrc.Parse(filepath.Join(usr.HomeDir, ".netrc"))
+	if err != nil {
+		return u, p, err
+	}
+	u = n.Machine("aperture.section.io").Get("login")
+	p = n.Machine("aperture.section.io").Get("password")
+	return u, p, err
+}
+
+func request(method string, url string, body io.Reader) (resp *http.Response, err error) {
+	client := &http.Client{
```

Comment:
Can we set the User-Agent header to at least specify section-cli, and ideally contain the binary version injected by the CI process? E.g.:

- [set the header](https://github.com/section-io/varnish-cli-bridge/blob/5fbc9bb62c73db6e2bf716e6b70f7d48a19d08b0/api.go#L33)
- [define the value from build-time values](https://github.com/section-io/varnish-cli-bridge/blob/5fbc9bb62c73db6e2bf716e6b70f7d48a19d08b0/main.go#L81)
- [inject the build-time values](https://github.com/section-io/varnish-cli-bridge/blob/7890759da5d61bbba69650c7602ac222fbc03dab/.travis.yml#L10)

Reply:
@jstangroome good call, those are great suggestions.

I'll add them to this PR before I merge.

Reply:
Fixed in e7e9968.